R/step-subset-arrange.R

```diff
@@ -24,8 +24,17 @@ arrange.dtplyr_step <- function(.data, ..., .by_group = FALSE) {
     return(.data)
   }
 
+  no_transmute <- all(map_lgl(dots, ~ is_symbol(.x) || is_call(.x, "-", 1)))
```

Comment:
To handle `arrange(-(a + b))`, need to make sure the arg to `-` is a symbol. Maybe like
```r
  is_simple_dot <- function(x) {
    is_symbol(x) || (is_call(x, "-", 1) && is_symbol(x[[2]]))
  }
  no_transmute <- all(map_lgl(dots, is_simple_dot))
```